hugegraph-mysql/src/main/java/com/baidu/hugegraph/backend/store/mysql/MysqlOptions.java

```diff
@@ -98,10 +98,20 @@ public static synchronized MysqlOptions instance() {
                     "false"
             );
 
+    public static final ConfigOption<Boolean> AUTO_RECONNECT =
+            new ConfigOption<>(
+                    "jdbc.auto_reconnect",
```

Comment:
prefer to rename to force_auto_reconnect, and set autoReconnect=true in MysqlSessions.buildUri() if force_auto_reconnect=true